iptables/table.go

```diff
@@ -214,6 +214,12 @@ type Table struct {
 	// to top-level chains.
 	insertMode string
 
+	// Record when we did our most recent updates and refreshes of the table.  We use these to
+	// calculate the next time we should force a refresh.
+	lastUpdateTime  time.Time
+	lastRefreshTime time.Time
```

Comment:
Can we rename 'refresh' to 'read' and 'update' to 'write'?  Until I had read more of the code below, I couldn't tell the difference between 'update' and 'refresh'.